pkg/servers/e2b/pause_resume_test.go

```diff
@@ -277,7 +280,7 @@ func TestResumeSandbox(t *testing.T) {
 						Type:   string(agentsv1alpha1.SandboxConditionReady),
 						Status: metav1.ConditionTrue,
 					})
-					client.ApiV1alpha1().Sandboxes(sbx.Namespace).UpdateStatus(context.Background(), sbx, metav1.UpdateOptions{})
+					_, _ = client.ApiV1alpha1().Sandboxes(sbx.Namespace).UpdateStatus(context.Background(), sbx, metav1.UpdateOptions{})
```

Comment:
can we move these test that require kind environment and operator installation to e2e tests? the unit test should use fakeclient and mocked data